### DISCUSSION TITLE
1.1.0

### DISCUSSION THREAD

**Discussion by rtivital** [mantinedev/mantine#87] [Releases]
## Release demos

[June 2021 changelog demos](https://mantine.dev/pages/changelog/#110--june-2021)

## New features and components
- [SegmentedControl](https://mantine.dev/core/segmented-control/)
- [Popover](https://mantine.dev/core/popover/)
- [Divider](https://mantine.dev/core/divider/)
- [Group](https://mantine.dev/core/group/)
- [Grid](https://mantine.dev/core/grid/)
- [use-debounced-value](https://mantine.dev/hooks/use-debounced-value/)

## API changes

- Prop typo fix in Checkbox component (~~intermediate~~ -> indeterminate)
- Hr component replaced with Divider to support more options
- ElementsGroup component replaced with Group to support all elements positioning
- Custom component is now supported by ActionIcon and MenuItem components
- Modal and Drawer components are wrapped with Portal by default
- LoadingOverlay now supports custom loaders
- @mantine/core components types are now exported from package, import it with Props suffix: `import { ButtonProps } from '@mantine/core'`

## Bug fixes and improvements

- Significant performance improvements with react-jss parameters memoization
- Dark theme improvements, all components now use more balanced transparent colors in dark theme
- Portal component now does not show warning during server side rendering
- @mantine/notifications – useNotifications hook now returns queue

<hr /><em>This discussion was created from the release <a href='https://github.com/mantinedev/mantine/releases/tag/1.1.0'>1.1.0</a>.</em>

**Comment by rtivital** [comment]
@cstrat Thanks for participation in this release, you've made a lot of good proposals, your feedback is very valuable

**Reply by cstrat** [reply]
Thank you for all the effort you've put into this.
I am glad that you don't mind all the posts/questions/reports 😄 

When I get a chance to I hope to get my head around typescript a bit more so I can try to contribute directly.

**Comment by juan-manuel-alberro** [comment]
I can see `v1.2.1` in the documentation but the latest release seems to be `v1.1.1`?
Also, the changelog is from `v1.1.0`.
Cheers.

**Reply by rtivital** [reply]
Yep, there was an incident during [esm modules testing](https://github.com/mantinedev/mantine/pull/90), no new things, just a mistake on my side
